### DISCUSSION TITLE
Which LoRaWAN version(s) is this library compatible with?

### DISCUSSION THREAD

**Discussion by HouzuoGuo** [mcci-catena/arduino-lorawan#224] [Q&A]
Hey! The Things Stack Community Edition seems to have defined about 7 LoRaWAN versions at this point - 1.0, 1.0.1, etc. Which version or versions is this library compatible with? Thanks!

**Comment by terrillmoore** [comment]
Tested with 1.0.3, and designed to 1.0.3. Other might work, but not tested.
